.pullapprove.yml

```diff
@@ -6,10 +6,11 @@ requirements:
 
 # Disallow approval of PRs still under development
 always_pending:
-  title_regex: 'WIP'
+  title_regex: 'WIP|RFC'
```

Comment:
@laijs I think you want parenthesis around that: `(WIP|RFC)`

Reply:
o, I should copy what @jodh-intel has said.

Reply:
@sameo updated, thanks,